plugins/modules/ipadnszone.py

```diff
@@ -352,6 +359,24 @@ def get_ipa_skip_nameserver_check(self):
         if not self.zone and self.ipa_params.skip_nameserver_check is not None:
             return self.ipa_params.skip_nameserver_check
 
+    @staticmethod
+    def _reverse_zone_name(ipaddress):
+        reverse_ip = dns.reversename.from_address(ipaddress)
+        try:
+            reverse_zone = dns.resolver.zone_for_name(reverse_ip)
+        except dns.resolver.NoNameservers:
+            # if reverse zone cannot be resolved, we'll use a sensible default.
+            reverse_zone = "in-addr.arpa."
```

Comment:
Since this string is used on 369 and 371 I would set it as a constant.

Reply:
done.

Reply:
Modified heuristics to match closer to IPA, and added comment on the function.